src/Sarif/Autogenerated/FailureLevel.cs

```diff
@@ -6,7 +6,7 @@
 namespace Microsoft.CodeAnalysis.Sarif
 {
     /// <summary>
-    /// A value that specifies the failure level of the result
```

Comment:
>failure [](start = 35, length = 7)

Why remove the word "failure"? If we think the type name `FailureLevel` is descriptive (and I do), then I think it's fair to specify that this is a "failure level" (as opposed to, for example, a confidence level).

Reply:
will fix, agree

---
In reply to: [256192667](https://github.com/Microsoft/sarif-sdk/pull/1280#discussion_r256192667) [](ancestors = 256192667)